mmcls/models/backbones/vgg.py

```diff
@@ -0,0 +1,185 @@
+import torch.nn as nn
+from mmcv.cnn import constant_init, kaiming_init, normal_init
+from mmcv.utils.parrots_wrapper import _BatchNorm
+
+from ..builder import BACKBONES
+from .base_backbone import BaseBackbone
+
+
+def conv3x3(in_planes, out_planes, dilation=1):
+    """3x3 convolution with padding."""
+    return nn.Conv2d(
+        in_planes,
+        out_planes,
+        kernel_size=3,
+        padding=dilation,
+        dilation=dilation)
+
+
+def make_vgg_layer(inplanes,
```

Comment:
better to change `inplanes` to `in_channels` and change `planes` to `out_channels` following the naming strategies of other backbones.